README.md

```diff
@@ -110,29 +120,25 @@ You can also help out by translating Finamp using our [weblate page](https://hos
 <img src=".">
 </details>
 
-</details>
 
 
 
 ## Shoutout
 - Thanks to all the [Contributors and Maintainer](https://github.com/jmshrv/finamp/graphs/contributors) who helped to make, fix and improve Finamp! Without you Finamp wouldn't be Finamp. ❤️
-- Thanks to the [Jellyfin Contributors](https://jellyfin.org/contribute/) without whom finamp wouldn't exists in the first place and thanks for making self-hosting and privacy easier!
-- Thanks to all the Developers who created and maintain packages finamp uses too!
-- And thanks to you for using Finamp!
+- Thanks to the [Jellyfin Contributors](https://jellyfin.org/contribute/) without whom Finamp wouldn't exists in the first place and thanks for making self-hosting and privacy easier!
+- Thanks to all the Developers who created and maintain packages Finamp uses!
+- And **you** for using Finamp!
```

Comment:
Where did the `thank` go? xD
Should be `And thank **you** for using Finamp!`

Reply:
Man, this kind of stuff happens to me way too offen